### PR TITLE
don't display custom colours in map name

### PR DESCRIPTION
**Pull request by Arkanic**
ahh yes ugly .replaceAll()

changes
```
[orange]cool map
```
to
```
cool map
```

### CODE REVIEW

**Comment by Arkanic** [comment]
i couldn't think of anything else to replace the text on the inside of the []

**Comment by Quezler** [comment]
Aww, that would make my

File: src/io/anuke/corebot/Messages.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public Messages(){
                         }else{
                             embed.addField(result.ip,
                             Strings.format("*{0}*\nPlayers: {1}\nMap: {2}\nWave: {3}\nVersion: {4}\nPing: {5}ms\n_\n_\n",
-                            result.host.replace("\\", "\\\\").replace("_", "\\_").replace("*", "\\*").replace("`", "\\`"), result.players, result.map.replace("\\", "\\\\").replace("_", "\\_").replace("*", "\\*").replace("`", "\\`"), result.wave, result.version, result.ping), false);
+                            result.host.replace("\\", "\\\\").replace("_", "\\_").replace("*", "\\*").replace("`", "\\`").replaceAll("\\[.*?\\]", ""), result.players, result.map.replace("\\", "\\\\").replace("_", "\\_").replace("*", "\\*").replace("`", "\\`"), result.wave, result.version, result.ping), false);
```
Comment (Anuken) [inline src/io/anuke/corebot/Messages.java]: This replaces the host name, not the map name.

Comment (Arkanic) [reply src/io/anuke/corebot/Messages.java]: oops

Comment (Arkanic) [reply src/io/anuke/corebot/Messages.java]: @Anuken fixed

**Comment by Arkanic** [comment]
@Quezler that's pretty cool, does it just do that specifically for coreBots ip?

**Comment by Quezler** [comment]
@Arkanic yea pretty much 😇 

**before & after**
Screen Shot 2019-11-04 at 08 40 48

**changes in the server's jar file**
Screen Shot 2019-11-04 at 08 41 14
Screen Shot 2019-11-04 at 08 41 02

@**Anuken**
for some (probably escaping-related) issue one of the "spider's legs" doesn't show in the ping but does in the server list, ima look into that somewhere today probably to find the difference 🤔 
Screen Shot 2019-11-04 at 08 44 10

**Comment by Quezler** [comment]
(oh and for the record, yes, i just publicly mentioned corebot's ip, but i doubt it's a big deal since it's easy to find, but do let me know if you'd like me to take down said image to make it a little harder than no effort at all for people to find it 😗)

**Comment by Arkanic** [comment]
@Quezler nice

**Pull request merged**